src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -151,28 +236,27 @@ AmendmentTableImpl::get (std::string const& name)
     return uint256 ();
 }
 
-AmendmentState*
-AmendmentTableImpl::addKnown (const char* amendmentID, const char* friendlyName,
-    bool veto)
+bool
+AmendmentTableImpl::addKnown (uint256 const& hash,
+                              std::string const& friendlyName,
+                              bool veto)
```

Comment:
We strongly recommend not using booleans for these sorts of purposes - because they're hard to audit at the call site.  You could accidentally put `addKnown ({id, name}, true);  // Accept amendment` and your reviewer might just nod eir head.

It's a little verbose but I think you'll come to love using `enum class` entries for this purpose.  In this case you'd have something like:

    enum class Veto {no, yes};
    bool AmendmentTableImpl::addKnown (AmendmentName const& name, Veto veto) //...
    
    // elsewhere:
    table.addKnown (name, Veto::no);